bookingBmodule.js

```diff
@@ -0,0 +1,117 @@
+import { default as requestPOST, } from './bookingCmodule.js';
```

Comment:
What you've done here with `default as requestPOST` is a useful way to import a named export with another name and the one it was exported as. It can be very useful for _named exports_.

For _default exports_ however, it provides no benefit over just writing the same code like this:

```js
import requestPOST from './bookingCmodule';
```